src/D2L.Bmx/D2L.Bmx.csproj

```diff
@@ -15,7 +15,7 @@
     Release builds (preview or real release) won't have git hash so they look cleaner.
     -->
     <IncludeSourceRevisionInInformationalVersion Condition="'$(Version)' != ''">false</IncludeSourceRevisionInInformationalVersion>
-    <Version Condition="'$(Version)' == ''">3.0.0-scratch</Version>
+    <Version Condition="'$(Version)' == ''">3.999.999-scratch</Version>
```

Comment:
make the dummy version number big, so the binary doesn't always prompt you to update to the latest release